dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/DefaultTrackerObjectsDeletionService.java

```diff
@@ -113,7 +113,11 @@ public TrackerTypeReport deleteEnrollments(List<String> enrollments) throws NotF
       TrackedEntity te = enrollment.getTrackedEntity();
       te.setLastUpdatedByUserInfo(userInfoSnapshot);
 
-      apiEnrollmentService.deleteEnrollment(enrollment);
+      enrollment.setStatus(EnrollmentStatus.CANCELLED);
+      enrollment.setDeleted(true);
+      enrollment.setLastUpdated(new Date());
+      manager.update(enrollment);
```

Comment:
Why are you not using `manager.delete` like we do for the other entities?

Reply:
I think this is too many steps here.
We can add the `delete` method to our `EnrollmentStore` and make it change the status and call the super.delete

Reply:
And then we should use manager.delete here and in the tests.